### DISCUSSION TITLE
Anyone having problems with JPG image orientation after upload?

### DISCUSSION THREAD

**Discussion by nzyeehaa** [datenstrom/community#478] [Ask a question] [answered]
having some problems with image orientation.

image taken on cell phone inserts exif tag from accelerometer re:orientaion
transfer image to computer, software accomodates exif tag automatically and shows image upright - but doesnt save the new orientation

upload image to yellow - it strips exif tag and image is upside down

https://www.lifewire.com/why-is-my-picture-turned-sideways-1701398
https://www.ryadel.com/en/change-image-orientation-iphone-andor-android-phone-pics-net-c/

**Comment by markseuffert** [comment, marked as answer]
Now the  [image extension](https://github.com/annaesvensson/yellow-image) supports resizing and reorienting uploaded images. This happens automatically when uploading large JPG images with the web browser, typically taken on a mobile phone.

**Comment by nzyeehaa** [comment]
wow, mark - thanks for your speedy response and fix

mobile phone exif tags now work and image is upright

thanks
